### PR TITLE
[update] calling FactoryBot module in requests/tokens_spec.rb

### PR DESCRIPTION
**Pull request by codescaptain**
## Why?
Always we calling FactoryBot module in the test file and I think It makes harder to reading code.

## What did I do?
I added Rspec confing add in `supports/dependencies/factory_bot.rb`. I read FactoryBot documentaion and I found this code :  [Setup](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#setup)

### Checklist 
- I run all tests about this PR (165 examples, 0 failures)

### CODE REVIEW

**Comment by nejdetkadir** [comment]
Thank you for your contribution 🙏🏻🚀

File: spec/supports/dependencies/factory_bot.rb [hunk 1]
```diff
@@ -1,4 +1,10 @@
 # frozen_string_literal: true
 
 require 'factory_bot'
-FactoryBot.find_definitions
+RSpec.configure do |config|
```
Comment (nejdetkadir) [inline spec/supports/dependencies/factory_bot.rb:4]: can you add this configuration to [here](https://github.com/nejdetkadir/devise-api/blob/main/spec/spec_helper.rb)?

Comment (nejdetkadir) [reply spec/supports/dependencies/factory_bot.rb:4]: you can delete `spec/supports/dependencies/factory_bot.rb`file

Comment (codescaptain) [reply spec/supports/dependencies/factory_bot.rb:4]: I believe that is correct, but the gem suggests adding the code to the factory_bot.rb file. @nejdetkadir

Comment (nejdetkadir) [reply spec/supports/dependencies/factory_bot.rb:4]: Okey, thank you for all 🙏🏻

**Review by nejdetkadir — APPROVED** [review]

**Pull request merged**